index.d.ts

```diff
@@ -0,0 +1,77 @@
+export interface JsonMLAttributes {
+  [key: string]: any // @TODO - can this be anything but a string? it technically SHOULD be,
+      // but I'm not sure if we want to be that strict. maybe it should be primitives at least?
+      // or maybe we just leave it alone?
+}
+
+export type JsonMLChildNode = string | JsonMLNode;
+
+export type JsonMLChildren = Array<JsonMLChildNode>;
```

Comment:
These two types seemed helpful. I thought it was useful to distinguish between a an element node (array) and a value, which could also be a string / text node.